db/migrate/20150824025314_add_author_id_to_record_edits.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorIdToRecordEdits < ActiveRecord::Migration
```

Comment:
Unless both of you have run these migrations and this'll be a pain in the butt to reconcile, I'd rather just rollback and add this column to the create_record_edits migration, rather than adding a new one in a separate file.